context/app/static/js/components/Markdown/style.js

```diff
@@ -0,0 +1,13 @@
+import styled from 'styled-components';
+import Paper from '@material-ui/core/Paper';
+
+const StyledPaper = styled(Paper)`
+  & img {
+    max-width: 100%;
+  }
+  padding: 30px 40px 30px 40px;
+`;
+// TODO: Copied and pasted the padding.
```

Comment:
From where?

Reply:
I don't remember... it's been copied and pasted a few times already!
```
$ grep '30px 40px 30px 40px' -r .
./components/Detail/ProvGraph/style.js:  padding: 30px 40px 30px 40px;
./components/Detail/Attribution/style.js:  padding: 30px 40px 30px 40px;
./components/Detail/Metadata/style.js:  padding: 30px 40px 30px 40px;
./components/Detail/Summary/style.js:  padding: 30px 40px 30px 40px;
```